packages/manager/apps/hub/src/dashboard/products-list/config.js

```diff
@@ -1,61 +1,15 @@
-import { find, get, head, isEmpty, map, startCase } from 'lodash-es';
-
+import { find, get, head } from 'lodash-es';
```

Comment:
Import lodash `find` is necessary ? (i've impression, native one is used here)

Reply:
Used here: https://github.com/ovh/manager/blob/dc00d786c263e49976c0db41d20a98196d3731cc/packages/manager/apps/hub/src/dashboard/products-list/config.js#L17

Reply:
USed here: https://github.com/ovh/manager/blob/dc00d786c263e49976c0db41d20a98196d3731cc/packages/manager/apps/hub/src/dashboard/products-list/config.js#L17